.github/workflows/release.yaml

```diff
@@ -89,44 +89,71 @@ jobs:
         env:
           GOOS: ${{ matrix.platform.os }}
           GOARCH: ${{ matrix.platform.arch }}
-          VERSION: ${{ needs.set-version.outputs.version }}
+          VERSION: ${{ github.event.release.tag_name }}
```

Comment:
This creates the version folder with the binaries, specifically changed this line so that the folder has the `v` prefix which will be uploaded to the cloud bucket.